src/Api/DataProvider/CoverCollectionDataProvider.php

```diff
@@ -36,12 +37,33 @@ public function getCollection(string $resourceClass, string $operationName = nul
         $isIdentifiers = $this->getIdentifiers($request);
 
         $query = $this->buildElasticQuery($identifierType, $isIdentifiers);
-        $searchResponse = $this->index->request('_search', \Elastica\Request::POST, $query->toArray());
-        $results = $searchResponse->getData();
+
+        // Note that we here don't uses the elastica request function to post the request to elasticsearch has we have
```

Comment:
" to elasticsearch has we have" => " to elasticsearch because we have"

Reply:
Fixed